build-logic-settings/update-develocity-plugin-version.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [ -z "$1" ]; then
+  echo "Usage: $0 <new-version>"
+  exit 1
+fi
+
+NEW_VERSION=$1
+
+FILES=(
+  "build-logic-commons/build-platform/build.gradle.kts"
+  "build-logic-settings/settings.gradle.kts"
+  "settings.gradle.kts"
+)
+
+for FILE in "${FILES[@]}"; do
+  if [ -f "$FILE" ]; then
+    sed -i.bak "s/com.gradle:develocity-gradle-plugin:[0-9]*\.[0-9]*\.[0-9]*/com.gradle:develocity-gradle-plugin:${NEW_VERSION}/" "$FILE"
```

Comment:
💅 you can remove `.bak` on both sed executions and then remove the `rm` below as well.